packages/core/src/external/athenahealth/index.ts

```diff
@@ -333,27 +373,27 @@ class AthenaHealthApi {
     const { log, debug } = out(
       `AthenaHealth create medication - cxId ${cxId} practiceId ${this.practiceId} patientId ${patientId} departmentId ${departmentId}`
     );
-    const medicationOptions = await this.searchForMedication({
-      cxId,
-      patientId,
-      medication: medication.medication,
-    });
-    if (medicationOptions.length === 0) throw new MetriportError("No medication options found");
-    const data = {
-      departmentid: this.stripDepartmentId(departmentId),
-      providernote: "Added via Metriport App",
-      unstructuredsig: "Metriport",
-      medicationid: `${medicationOptions[0]?.medicationid}`,
-      hidden: "false",
-      startdate: this.formatDate(medication.statement?.effectivePeriod?.start),
-      stopdate: this.formatDate(medication.statement?.effectivePeriod?.end),
-      stopreason: undefined,
-      patientnote: undefined,
-      THIRDPARTYUSERNAME: undefined,
-      PATIENTFACINGCALL: undefined,
-    };
     const chartMedicationUrl = `/chart/${this.stripPatientId(patientId)}/medications`;
     try {
+      const medicationOptions = await this.searchForMedication({
+        cxId,
+        patientId,
+        medication: medication.medication,
+      });
+      if (medicationOptions.length === 0) throw new MetriportError("No medication options found");
+      const data = {
+        departmentid: this.stripDepartmentId(departmentId),
+        providernote: "Added via Metriport App",
+        unstructuredsig: "Metriport",
+        medicationid: `${medicationOptions[0]?.medicationid}`,
```

Comment:
Is this expected to get the string `"undefined"`?

Reply:
no... need to adjust typing

Reply:
Done.